webpack/stories/docs/addingDependencies.md

```diff
@@ -0,0 +1,21 @@
+
+# Adding or updating NPM dependencies
+
+The easiest way of adding dependencies into [package.json](https://github.com/theforeman/foreman/blob/develop/package.json) is executing `npm install` with `--save` or `--save-dev` for devel dependencies:
+```
+npm install --save <pkg>@<version>
+npm install --save-dev <pkg>@<version>
+```
```

Comment:
might add --no-optional to reduce node modules size (e.g. if you dont use phatomjs)

Reply:
I added a mention here: https://github.com/theforeman/foreman/pull/5883/files#diff-de4d6b944c9164c2bc2c69ecdf26d018R25